kura/distrib/src/main/resources/aarch64-nn/recover_default_config.init

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+echo "This script performs rollback to factory default configuration."
+echo "If you run this script, all configuration snapshots, except snapshot_0, will be removed."
+read -r -p "Are you sure? [y/N] " response
+case $response in
+    [yY][eE][sS]|[yY]) 
+        script_dir=$(dirname $0)
+	data_dir=$script_dir/data
+	snapshots_dir=$data_dir/snapshots
+	dflt_data_dir=$script_dir/.data
+	
+	for snapshot_file in "$snapshots_dir"/*
+	do
+	    if [ $snapshot_file != $snapshots_dir"/snapshot_0.xml" ]; then 
+  		echo Removing the $snapshot_file configuration file ...
+		rm -f $snapshot_file
+		if [ "$?" = "0" ]; then
+            	    echo "  done"
+	        fi
+	    fi
+	done
+	
+	echo Copying the $dflt_data_dir/dhcpd-eth0.conf file to the /etc directory ...
```

Comment:
Same here: it's an NN profile and the recover file should not touch networking related files.

Reply:
About all the NN profile comments: I started with a non NN profile and then change to a NN. I didn't removed all this. I will fix them.